### PR TITLE
Add documentation for NewEnterpriseClient

### PR DESCRIPTION
**Pull request by jlamb1**
adds a simple example how to connect to an enterprise GitHub instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,11 @@ func main() {
 }
 ```
 
+If using GitHub Enterprise, use `NewEnterpriseClient` :
```
Comment (dmitshur) [inline README.md]: Please make a few changes:

- If using ... → If you are using ...
- \`NewEnterpriseClient\` → \[\`githubv4.NewEnterpriseClient\`\]\(https\://godoc.org/github.com/shurcooL/githubv4#NewEnterpriseClient\)
- Remove the space before the colon.
- Add a blank line after the colon.

File: README.md [hunk 2]
```diff
@@ -46,6 +46,11 @@ func main() {
...
+```Go
+client := githubv4.NewEnterpriseClient(os.Getenv("GITHUB_ENDPOINT"), httpClient)
```
Comment (dmitshur) [inline README.md]: Add after this line:

```
// Use client...
```

For consistency with the code above.

**Comment by BlackVegetable** [comment]
If I finished this PR with the changes you've suggested, would you merge it?

**Comment by dmitshur** [comment]
@BlackVegetable Yes, thanks.

**Comment by jlamb1** [comment]
apologies for the delay @dmitshur, i've added a commit to address your review here.

**Pull request merged**